tests/probes_test.go

```diff
@@ -47,11 +47,15 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			},
 		}
 		table.DescribeTable("should succeed", func(readinessProbe *v12.Probe, serverStarter func(vmi *v12.VirtualMachineInstance, port int)) {
+			if tests.IsRunningOnKindInfra() && readinessProbe == httpProbe {
+				Skip("Skip http probe test till PR https://github.com/kubevirt/kubevirt/pull/3224 is merged")
+			}
 			By("Specifying a VMI with a readiness probe")
 			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
 			vmi.Spec.ReadinessProbe = readinessProbe
 			vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
 			Expect(err).ToNot(HaveOccurred())
+			tests.WaitUntilVMIReady(vmi, tests.LoggedInCirrosExpecter)
```

Comment:
Done, it indeed was added to configure the ipv6. But the tests are passing without it so removed it.

Reply:
I'm puzzled as to how 0_o

I actually meant the `tests.WaitUntilVMIReady(vmi, tests.LoggedInCirrosExpecter)` clause to be kept, and the `tests.WaitForSuccessfulVMIStartIgnoreWarnings(vmi)` to be dropped (since we need to configure the ip address via console, afaiu).

I'll try to understand that a bit better, but so far, I'm almost sure configuring the ipv6 address via console **is needed** for the http probe test (the one that is currently skipped).